src/actions/tokens.example.js

```diff
@@ -1,11 +0,0 @@
-/**
```

Comment:
Is this what you were saying you had to remove for the geocoding to work? I don't see how this could affect that since it is purely for documentation purposes and isn't supposed to be called anywhere. I'd rather leave it in if possible, but if you think it is what caused the issue I am fine with removing it.